crates/hir_ty/src/lib.rs

```diff
@@ -566,7 +566,7 @@ mod tests {
 
                     stmt_str
                 }
-                hir::Stmt::ExprStmt {
+                hir::Stmt::Expr {
```

Comment:
**Security Issue**: The change from 'hir::Stmt::ExprStmt' to 'hir::Stmt::Expr' might introduce inconsistencies in how expressions are handled, potentially affecting security checks or validations applied to these expressions. <br> **Fix**: Review the handling of 'hir::Stmt::Expr' to ensure that all necessary security checks and validations are correctly applied. <br> **Code Suggestion**: 
 ```
 -                hir::Stmt::ExprStmt {
 +                hir::Stmt::ExprStmt {
 ```